common/components/Header/components/TabsOptions.jsx

```diff
@@ -23,7 +23,8 @@ const tabs = [
         link: 'view-wallet'
     },
     {
-        name: 'Help'
+        name: 'Help',
```

Comment:
So re:RR, we should probably switch to v4. No specific reason besides forward compatibility.